docs/projects/ad3552r_evb/index.rst

```diff
@@ -150,8 +153,8 @@ the HDL repository, and then build the project as follows:
 
 .. shell::
 
-   $cd hdl/projects/ad3552r_evb/zed
-   $make
+   user@analog:~$ cd hdl/projects/ad35xxr_evb/zed
+   user@analog:~/hdl/projects/ad35xxr_evb/zed$ make
```

Comment:
Please respect the syntax: https://analogdevicesinc.github.io/doctools/docs_guidelines.html#shell-directive
this directive is intended to provide a consistent shell snippets to the user, and it makes only the actual command selectable.